photo-points/Controllers/AdminController.cs

```diff
@@ -17,14 +17,14 @@ public class AdminController : Controller
     {
 
 
-        //private PhotoDataContext _pdc;
+        private PhotoDataContext _pdc;
```

Comment:
replace the Data Context with private instance variable of type IAdminReviewService